pom.xml

```diff
@@ -8,10 +8,10 @@
 		<relativePath /> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>it.eng.idsa</groupId>
-	<artifactId>market4.0-data-app</artifactId>
+	<artifactId>true-connector-data-app</artifactId>
 	<version>0.2.5-SNAPSHOT</version>
-	<name>market4.0-data-app</name>
-	<description>The project for market4.0-data-app</description>
+	<name>true-connector-data-app</name>
+	<description>TRUE Connector data app</description>
```

Comment:
TRUE Connector or TRUEConnector for description?
I would also add Basic DataApp (like it is in readme)